docs/modules/ROOT/pages/cops_rspec_factorybot.adoc

```diff
@@ -1,5 +1,78 @@
 = RSpec/FactoryBot
 
+== RSpec/FactoryBot/AssociationStyle
+
+|===
+| Enabled by default | Safe | Supports autocorrection | Version Added | Version Changed
+
+| Pending
+| No
+| Yes (Unsafe)
+| <<next>>
+| -
+|===
+
+Use consistent style to define associations.
+
+=== Safety
+
+This cop may cause false-positives in `EnforcedStyle: explicit`
+case. It recognizes any method call that has no arguments as an
+implicit association but it might be a user-defined trait call.
+
+=== Examples
+
+==== EnforcedStyle: implicit (default)
+
+[source,ruby]
+----
+# bad
+association :user
+
+# good
+user
+----
+
+==== EnforcedStyle: explicit
+
+[source,ruby]
+----
+# bad
+user
+
+# good
+association :user
+
+# good (defined in NonImplicitAssociationMethodNames)
+skip_create
+----
+
+=== Configurable attributes
+
+|===
+| Name | Default value | Configurable values
+
+| inherit_mode
+| `{"merge"=>["NonImplicitAssociationMethodNames"]}`
```

Comment:
Unrelated to this PR. Seems like a doc generation problem, this shouldn't be here.